src/components/Header.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onInputChange: (newTitle: string) => void,
+  inputValue: string,
+  setInputValue: React.Dispatch<React.SetStateAction<string>>,
+  isInputFieldDisabled: boolean,
+  onHandleChangeCompleted: () => void,
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  onInputChange,
+  inputValue,
+  setInputValue,
+  isInputFieldDisabled,
+  onHandleChangeCompleted,
+}) => {
+  const noCompletedTodos = todos?.length
+    ? todos.every((todo) => todo.completed)
+    : false;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every - read how this method works and correct the error

Reply:
Not fixed